### DISCUSSION TITLE
XYChart multi line with hover on each line

### DISCUSSION THREAD

**Discussion by monski** [airbnb/visx#1280] [Q&A] [answered]
Hi visx team,

I'm trying to build chart where a single data points can only selected on hover.
For example on the image if i hover on "line a", it gets thick and data points (AnimatedGlyphSeries) is revealed. is it possible? 

thanks in advance

![Screen Shot 2021-07-16 at 9 31 20 PM](https://user-images.githubusercontent.com/2107437/125955949-4ee1bdb7-b6a8-4ec1-af8d-41a9d3885593.png)

**Comment by williaster** [comment, marked as answer]
Hey @monski 👋  thanks for checking out `visx`. If I understand what you want to do, I think you could probably do this with simple react state management. Here's a [demo](https://codesandbox.io/s/glyph-series-on-hover-du5r4) using button clicks instead of hovers but you should be able to easily modify for your use

![Jul-21-2021 13-20-23](https://user-images.githubusercontent.com/4496521/126554733-9d91a85c-8c34-480f-86e6-74707d847b3a.gif)

**Reply by monski** [reply]
thank you so much @williaster !
